src/lib/Paths.js

```diff
@@ -1,6 +1,7 @@
-import { distance, fuzzyEqual } from "./PathHelpers"
+import { distance, fuzzyEqual } from "./PathHelpers";
+import { Path } from "./Path";
 
-export const DEFAULT_TIME_IMPRECISON = 0; // integer seconds
+export const DEFAULT_TIME_IMPRECISON = 30; // integer seconds
 export const DEFAULT_LOCATION_IMPRECISON = 0; // decimal degrees: 1.0° ~ 111km, 0.01° ~ 1.11km, 0.00001° ~ 1.11m
```

Comment:
Shouldn't we also add a default for the location, like `0.0001`? (11.1m)

Reply:
Fixed